src/components/ClickableStyle/ClickableStyle.module.css

```diff
@@ -494,9 +494,10 @@
 
 /**
  * Needs `:where` pseudoclass to reduce specificity so class overrides
- * for margin can happen where used.
+ * for margin can happen where used, but needs to be specific enough to override preflight.
  */
-:where(.clickable-style--link) {
+a:where(.clickable-style--link),
+button:where(.clickable-style--link) {
```

Comment:
the two element selectors here match the tags that can be used in the `as` prop on `ClickableStyle`, correct?

Reply:
Yup!